contrib/googletest/template.py

```diff
@@ -0,0 +1,32 @@
+pkgname = "googletest"
+pkgver = "1.14.0"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf",
+]
+pkgdesc = "Google Testing and Mocking Framework"
+maintainer = "ttyyls <contact@behri.org>"
+license = "BSD-3-Clause"
+url = "https://google.github.io/googletest"
+source = (
+    f"https://github.com/google/googletest/archive/refs/tags/v{pkgver}.tar.gz"
+)
+sha256 = "8ad598c73ad796e0d8280b082cebd82a630d73e73cd3c70057938a6501bba5d7"
+options = ["empty"]
+
+
+def post_install(self):
+    self.install_license("LICENSE")
+
+
+@subpackage("googletest-devel-static")
+def _static(self):
+    return self.default_static()
```

Comment:
for reference, the default-devel also splits -devel-static, you never need to declare this normally